assets/src/components/NavigationType/NavigationType.js

```diff
@@ -6,16 +7,22 @@ const NAVIGATION_TYPE_MINIMAL = 'minimal';
 
 /**
  * Navigation type selector
+ *
+ * @param  root0
```

Comment:
Same comment as earlier for these `root0` 😬

Reply:
😅